scripts/commands/updateApiDocs.ts

```diff
@@ -239,7 +239,8 @@ async function convertHtmlToMarkdown(
   }
 
   console.log("Generating version file");
-  const pkg_json = { name: pkg.name, version: pkg.version };
+  const type = pkg.historical ? "historical" : "latest";
+  const pkg_json = { name: pkg.name, version: pkg.version, type: type };
```

Comment:
You can use a nifty JS feature:

```suggestion
  const pkg_json = { name: pkg.name, version: pkg.version, type };
```